app/helpers/rad_common/application_helper.rb

```diff
@@ -179,6 +179,10 @@ def verify_manually_create_users
       raise RadIntermittentException
     end
 
+    def nav_container_class
+      @marketing_site ?  'container' : 'container-fluid'
```

Comment:
Not sure if I agree with this one, we set this to true so its easy to use it here. We do the same thing in navigation layout. The alternative is we make it a helper method on the application controller so that we can access both in the view and controller

Reply:
I like the rule in general, but doesn't make a ton of sense here and we have bigger fish to fry, suggest we ignore this and move on for now